src/main/js/screens/app/screens/quizes/QuizForm.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import {
+  Alert, Form, FormGroup, FormControl, Button,
+  ButtonGroup, Panel
+} from 'react-bootstrap';
+
+import quizService from '../../../../shared/services/quizService';
+
+class QuizForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      description: '',
+      error: null,
+    };
+
+    this.onInputChange = this.onInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const obj = { name: this.state.name, description: this.state.description };
+    quizService().create(obj).then((response) => {
+      if (response.status === 201) {
+        this.props.callParent(true);
+      } else {
+        throw new Error();
+      }
+    }).catch((err) => {
+      this.setState({
+        error: 'Något fick fel',
+      });
+    });
+  }
+
```

Comment:
I like this ⭐️ 

Perhaps, in order to clarify WHEN errorMsg is shown, move condition checking to render, i.e. `{this.state.error != null ? renderErrorMsg : null}` instead of `{renderErrorMsg()}`

Reply:
good idea!
✅